data/core/tips.json

```diff
@@ -288,5 +288,40 @@
     "type": "snippet",
     "category": "tip",
     "text": [ "Don't be afraid to make the game easier on yourself.  Worlds can be made to spawn with more resources." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don’t get caught with your pants down.  There’s no toilet paper anymore anyway." ]
```

Comment:
`Don’t...`. What is the symbol here between `n` and `t`? Shouldn't it be `'`? Also for `There’s...`.